slo-libsonnet/errorburn.libsonnet

```diff
@@ -0,0 +1,88 @@
+local errors = import 'errors.libsonnet';
+
+{
+  errorburn(param):: {
+    local slo = {
+      metric: 'http_requests_total',
```

Comment:
I think I'd prefer if one has to specify these, in the style of the jsonnet code here: https://landing.google.com/sre/workbook/chapters/configuration-specifics/

(search for the headline "Integrating the Configuration Language")

Reply:
Updated. :relaxed: